cdk/lib/snyk-tag-monitor.ts

```diff
@@ -1,41 +1,51 @@
-import type { GuStackProps } from "@guardian/cdk/lib/constructs/core";
-import { GuStack } from "@guardian/cdk/lib/constructs/core";
-import type {GuScheduledLambdaProps} from "@guardian/cdk/lib/patterns"
-import {GuScheduledLambda} from "@guardian/cdk/lib/patterns"
-import type { App} from "aws-cdk-lib";
-import { Duration } from "aws-cdk-lib";
-import { Schedule } from "aws-cdk-lib/aws-events";
-import { Runtime } from "aws-cdk-lib/aws-lambda";
-import {Secret} from "aws-cdk-lib/aws-secretsmanager";
-import { Topic } from "aws-cdk-lib/aws-sns";
-import { EmailSubscription } from "aws-cdk-lib/aws-sns-subscriptions";
-
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import type { GuScheduledLambdaProps } from '@guardian/cdk/lib/patterns';
+import { GuScheduledLambda } from '@guardian/cdk/lib/patterns';
+import type { App } from 'aws-cdk-lib';
+import { Duration } from 'aws-cdk-lib';
+import { Schedule } from 'aws-cdk-lib/aws-events';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
+import { Topic } from 'aws-cdk-lib/aws-sns';
+import { EmailSubscription } from 'aws-cdk-lib/aws-sns-subscriptions';
 
 export class SnykTagMonitor extends GuStack {
-  constructor(scope: App, id: string, props: GuStackProps) {
-    super(scope, id, props);
+	constructor(scope: App, id: string, props: GuStackProps) {
+		super(scope, id, props);
+
+		const app = 'snyk-tag-monitor';
 
-    const app = "snyk-tag-monitor"
+		const topic = new Topic(this, `${app}-topic`);
+		topic.addSubscription(
+			new EmailSubscription('devx.security@guardian.co.uk'),
+		);
 
-    const topic = new Topic(this, `${app}-topic`)
-    topic.addSubscription(new EmailSubscription("devx.security@guardian.co.uk"))
-    const lambdaProps: GuScheduledLambdaProps = {
-      rules: [{ schedule: Schedule.rate(Duration.days(2))}],
-      monitoringConfiguration: {
-        toleratedErrorPercentage: 50,
-        snsTopicName: topic.topicName,
-      },
-      runtime: Runtime.PYTHON_3_9,
-      handler: "main.hello",
-      app,
-      fileName: `${app}.zip` 
-    }
+		const snykApiKey = new Secret(this, 'snykApiKey');
```

Comment:
Why are these secrets and not SSM parameters of type secure string?

Reply:
A secure string ssm parameter is more typical, however it's nice to experiment... Ultimately, I think the [lambda layer for config pattern](https://github.com/guardian/ssm-to-env) is the best answer here, with the parameter being created outside of this stack, and the application simply reading the value from env vars. I don't think we have a complete story/process for this yet though?

Is this blocking, or can merge as is, and look at how we initialise and handle secrets separately?